website/docs/docs/core/connect-data-platform/profiles.yml.md

```diff
@@ -31,6 +31,10 @@ This section identifies the parts of your `profiles.yml` that aren't specific to
   [fail_fast](/reference/global-configs/failing-fast): <true | false>
   [use_experimental_parser](/reference/global-configs/parsing): <true | false>
   [static_parser](/reference/global-configs/parsing): <true | false>
+  [cache_selected_only](/reference/global-configs/cache): <true | false>
+  [no_print](/reference/global-configs/print-output#suppress-print-messages-in-stdout): <true | false>
```

Comment:
I tried out specifying `no_print:` within `profiles.yml`, and it didn't have any effect. So we should remove it here.
```suggestion
```

Reply:
Created this issue as a result: https://github.com/dbt-labs/docs.getdbt.com/issues/4759